bower.json

```diff
@@ -1,14 +1,14 @@
 {
   "author": "Mozilla",
   "name": "marketplace-core-modules",
-  "version": "2.1.0",
+  "version": "3.0.0",
   "ignore": [
     "bower.json",
     "LICENSE",
     "README.md"
   ],
   "dependencies": {
-    "jquery": "2.0.2",
+    "marketplace-jquery": "2.0.2",
```

Comment:
Is this just a trimmed down build? Seems silly that we have our own bower module for this but I guess we may as well.